<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1346</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
    background-image: url(&quot;:/rsc/C:/Users/somai/Downloads/imgpro (1).jpg&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>52</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
color: rgb(136, 143, 232); 
font-size:20px;
text-align: center;
font-weight:Bold;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>52</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
color: rgb(136, 143, 232); 
font-size:20px;
text-align: center;
font-weight:Bold;</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="crop">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>620</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Crop</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="color">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>349</y>
      <width>231</width>
      <height>75</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>White Balance</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>180</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Left Rotate</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="blur">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>260</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">   color: rgb(136, 143, 232); 
font-size:35px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Blur Image</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="flip">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>533</y>
      <width>231</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Flip Image</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>52</y>
      <width>171</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
color: rgb(136, 143, 232); 
font-size:20px;
text-align: center;
font-weight:Bold;</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>300</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
   color: rgb(136, 143, 232); 
font-size:35px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>    Add an Image </string>
    </property>
   </widget>
   <widget class="QLabel" name="img2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>300</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Semibold</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
   color: rgb(136, 143, 232); 
font-size:35px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>    Edited Image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gray">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>260</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">   color: rgb(136, 143, 232); 
font-size:35px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Gray Scale</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rightRotate">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>180</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Right Rotate</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="resize">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>620</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Resize</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>630</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="sliderPosition">
     <number>0</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="sharp">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>349</y>
      <width>231</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Sharp</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="enlarge">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>533</y>
      <width>231</width>
      <height>75</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Enlarge</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="brightness">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>440</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="contrast">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>440</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(233, 235, 251);
    color: rgb(136, 143, 232); 
font-size:35px;
</string>
    </property>
    <property name="text">
     <string>Contrast</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="brightnessSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>670</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="contrastSlider">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>650</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="pageStep">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   color: rgb(136, 143, 232); 
font-size:20px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>Before</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">   color: rgb(136, 143, 232); 
font-size:20px;
text-align: center;</string>
    </property>
    <property name="text">
     <string>After</string>
    </property>
   </widget>
   <widget class="QFrame" name="cropFrame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>101</width>
      <height>131</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>350</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid red;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
